modules/macros/src/main/scala/ru/tinkoff/tschema/macros/MakerMacro.scala

```diff
@@ -55,6 +57,8 @@ class MakerMacro(val c: blackbox.Context) extends ShapelessMacros with Singleton
     val dsl       = constructDslTree(defT, Monoid.empty[PrefixInfo[Type]])
     val wholeTree = new RouteTreeMaker(impl).makeRouteTree(ft, dsl, input.tree)
 
+    findMonadInstanceOrAbort(ft)
```

Comment:
How would this work for the akka-http module without dependency on cats?

Reply:
I assume it won't typecheck the `implicitlyTree` because Akka uses `Skip[A] = Any` which has been treated in a special way at this line:
https://github.com/TinkoffCreditSystems/typed-schema/pull/314/files#diff-a577d3df9c2facf7b7db4047f1596557R267

Although I didn't check it manually.